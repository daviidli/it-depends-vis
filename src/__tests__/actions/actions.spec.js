import {
	setError, setRepo, setData, setSelectedFile, setRange, setCommits, setStartCommit, setEndCommit
} from '../../actions/actions';

describe('actions', () => {
	it('should create setRepo action', () => expect(setRepo('github.com/somerepo')).toMatchSnapshot());

	it('should create setData action', () => expect(setData({ data: 'someData' })).toMatchSnapshot());

	it('should create setSelectedFile action', () => expect(setSelectedFile('file1')).toMatchSnapshot());

	it('should create setRange action', () => expect(setRange([0, 100])).toMatchSnapshot());

	it('should create setCommits action', () => expect(setCommits(['commit1', 'commit2'])).toMatchSnapshot());

	it('should create setStartCommit action', () => expect(setStartCommit('firstCommit')).toMatchSnapshot());

	it('should create setEndCommit action', () => expect(setEndCommit('lastCommit')).toMatchSnapshot());
});
