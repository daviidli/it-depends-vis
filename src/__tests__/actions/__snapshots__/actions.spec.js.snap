// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions should create setCommits action 1`] = `
Object {
  "commits": Array [
    "commit1",
    "commit2",
  ],
  "type": "SET_COMMITS",
}
`;

exports[`actions should create setData action 1`] = `
Object {
  "data": Object {
    "data": "someData",
  },
  "type": "SET_DATA",
}
`;

exports[`actions should create setEndCommit action 1`] = `
Object {
  "endCommit": "lastCommit",
  "type": "SET_END_COMMIT",
}
`;

exports[`actions should create setRange action 1`] = `
Object {
  "range": Array [
    0,
    100,
  ],
  "type": "SET_RANGE",
}
`;

exports[`actions should create setRepo action 1`] = `
Object {
  "repo": "github.com/somerepo",
  "type": "SET_REPO",
}
`;

exports[`actions should create setSelectedFile action 1`] = `
Object {
  "filename": "file1",
  "type": "SET_SELECTED_FILE",
}
`;

exports[`actions should create setStartCommit action 1`] = `
Object {
  "startCommit": "firstCommit",
  "type": "SET_START_COMMIT",
}
`;
